### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by ferricoxide**
Added two items to FAQ (details in commit messages).

@lorengordon: I'm a little iffy on the link-methodology - particularly with respect to anchor-links (as in commit 9a08758). Was mostly aping the link-style in the `What do I do if watchmaker failed to install` FAQ-item. Is there a good reference around MD-optimization for readthedocs rendering? Seems like within-GitHub linking is sacrificed in favor of within-ReadTheDocs linking.

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ the output of a failed installation. Usually, the output points pretty clearly
 at the source of the problem. Watchmaker can be re-installed over itself with
 no problem, so once the root cause is resolved, simply re-install watchmaker.
 
+## Why does the watchmaker install fail if my system is FIPS enabled?
+
+This is primarily a question for Red Hat. As of this writing, the `pip` utility in all Red Hat releases up through 7.4.1708, default to looking for pypi packages signed with MD5 signatures. If you've enabled FIPS (or are using a build that has FIPS pre-enabled), MD5 is disabled in the kernel (due to being a weak hashing-method). You can either disable FIPS (not recommended) or explicitly tell fips to use a differnt signature-index. The latter is detailed in the Linux section of the :doc:`usage <usage#linux>` document.
```
Comment (lorengordon) [inline docs/faq.md]: "tell fips" i think you mean "tell `pip`"?

Comment (ferricoxide) [reply docs/faq.md]: Dammit!

Don't FAQ and flexeril. 

Gimme a sec

File: docs/faq.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ the output of a failed installation. Usually, the output points pretty clearly
 at the source of the problem. Watchmaker can be re-installed over itself with
 no problem, so once the root cause is resolved, simply re-install watchmaker.
 
+## Why does the watchmaker install fail if my system is FIPS enabled?
+
+This is primarily a question for Red Hat (and derived distributions). As of this writing, the `pip` utility in all Red Hat releases up through 7.4.1708, default to looking for pypi packages signed with MD5 signatures. If you've enabled FIPS (or are using a build that has FIPS pre-enabled), MD5 is disabled in the kernel (due to being a weak hashing-method). You can either disable FIPS (not recommended) or explicitly force `pip` to use a different signature-index. The latter is detailed in the Linux section of the :doc:`usage <usage#linux>` document.
```
Comment (lorengordon) [inline docs/faq.md]: Unfortunately it seems can't currently link to an anchor directly, the way the docs are generated. Need to just do `usage <usage>` for now.

File: docs/faq.md (hunk 1 above)
Comment (ferricoxide) [reply docs/faq.md]: Updated.

File: docs/faq.md (hunk 2 above)
Comment (ferricoxide) [reply docs/faq.md]: Blergh. That's sucktacular. :-1:

File: docs/faq.md [hunk 3]
```diff
@@ -44,3 +48,12 @@ label may not be related to the error you are encountering.
 
 Watchmaker is supported on RedHat 7 and CentOS 7. However, there may be issues
 intermittently with the salt formulas as they are actively being developed.
+
+## Can I use watchmaker to toggle my RedHat/Centos host's FIPS mode?
+
+Yes, indirectly. Because watchmaker implements most of its functionality via [SaltStack](https://saltstack.com) modules, you can directly-used the underlying SaltStack functionality to effect the dsired change. This is done from the commandline - as root - by executing:
```
Comment (lorengordon) [inline docs/faq.md]: used -> use
dsired -> desired

Comment (ferricoxide) [reply docs/faq.md]: I really need to see about linking my vim vs. spel

File: docs/faq.md (hunk 2 above)
Comment (lorengordon) [reply docs/faq.md]: it's just the way it's rendered from markdown to rst to html. md -> html, no prob. or rst -> html, no prob. but md -> rst is a bit wonky. the recommonmark package from read-the-docs that currently handles the conversion has not been getting a lot of love for a while now.

**Pull request merged**